docs/api-reference.md

```diff
@@ -51,7 +51,15 @@ Similar to connection-builder but is used to build the JDBC properties file. For
 {"server" : "myserver.somewhere.net", "username" : "myusername", "password" : "mypassword"}
 ```
 
-**Return:** vector of formatted key=value pairs that will be written to the properties file
+**Return:** dictionary or vector
```

Comment:
just for the sake of clarity, do we want to also include a sample js object that might be returned?

Reply:
The postgres jdbc sample has been updated to return the dictionary